cmd/sfe/main.go

```diff
@@ -76,6 +77,21 @@ type Config struct {
 			// in this file must be identical to those in the RA.
 			Defaults string `validate:"required_with=Redis"`
 		}
+
+		// OverridesImporter configures the periodic import of approved rate
+		// limit override requests from Zendesk.
+		OverridesImporter struct {
+			// Mode controls which tickets are processed. Valid values are:
+			//   - "all": process all tickets
+			//   - "even": process only tickets with even IDs
+			//   - "odd": process only tickets with odd IDs
+			// If unspecified or empty, defaults to "all".
+			Mode string `validate:"omitempty,required_with=Interval,oneof=all even odd"`
```

Comment:
Nit: Implementing this as two "M of N" settings (with the same underlying modulo bucketing logic) is pretty simple and would allow a lot more flexibility. But this all/even/odd mode is already implemented and will work fine for us, so NBD.